.storybook/themeableAttributes.js

```diff
@@ -18,7 +18,6 @@
 
 /* @flow */
 
-import { themeAttributes as accordionThemeAttributes } from '../packages/bpk-component-accordion';
```

Comment:
is this change related to update semantic tokens?

Reply:
Not exactly. But we use these theming for `hover` and `active` states which we don't need in the accordion component as they will all have the same colours. So we can just get rid of them. It's more like cleaning up the code as we go